randovania/games/cave_story/logic_database/Ruined Egg Corridor.txt

```diff
@@ -41,7 +45,9 @@ Extra - starting_script: <TRA0049:0099:0007:0006
   * Layers: default
   > Door to Egg Observation Room? (East)
       Any of the following:
+          # Kill dragon
           Has Weapon
+          # Fly over dragon
           Pacifist Strats (Beginner) and Has Flight
           # Damage boost off the dragon
```

Comment:
see above

Reply:
this one however is prob worth a TODO yeah